site/downloads.markdown

```diff
@@ -9,168 +9,92 @@ isDownloads: true
 This page describes the installation of the Haskell toolchain, which consists of the following tools:
 
 *   [GHC](https://www.haskell.org/ghc/): the Glasgow Haskell Compiler
-
 *   [cabal-install](https://cabal.readthedocs.io): the Cabal installation tool for managing Haskell software
-
 *   [Stack](https://docs.haskellstack.org): a cross-platform program for developing Haskell projects
-
 *   [haskell-language-server](https://github.com/haskell/haskell-language-server) (optional): A language server for developers to integrate with their editor/IDE
 
-## Installation instructions { #ghcup }
+## Recommended installation instructions
 
 *for Linux, macOS, FreeBSD, Windows or WSL2*
 
 1. Install GHC, cabal-install and haskell-language-server via [GHCup](https://www.haskell.org/ghcup/)
-2. To install Stack, follow the instructions [here](https://docs.haskellstack.org/en/stable/install_and_upgrade/) *(N.B. Stack does not support FreeBSD)*
+2. To install Stack, follow the [Stack installation guide](https://docs.haskellstack.org/en/stable/install_and_upgrade/)
```

Comment:
Would it make sense for these to be bullet points rather than a numbered list? The numbered list implies to me that you need to follow each step in order, rather than having a choice of install method.

Reply:
Yes, that's a good point. However, the numbered list predates this PR so I suggest we leave that for a separate PR.